Cargo.toml

```diff
@@ -46,7 +48,7 @@ serde_json = "1.0"
 serde_yaml = "0.9"
 sha2 = "0.10"
 snafu = { version = "0.8", features = ["futures"] }
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "stackable-operator-0.74.0" }
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "stackable-operator-0.91.1", default-features = false }
```

Comment:
Just curious, what happens when we bump to 0.92.0?

My understanding was that dependencies can use whatever edition they want and it doesn't change the edition used here.

Reply:
Yes, I think that's how it should work. But I vaguely remember there being a different issue. Let me try it though.